### DISCUSSION TITLE
스카이박스 움직임 구현

### DISCUSSION THREAD

**Discussion by esang28** [naverz/zepeto-studio-kor#550] [5. Q&A]
제페토 월드 패키지 버전: 1.1.5
개발환경: 윈도우

유니티 초보에 제페토 엔진은 처음 사용해봅니다. 제페토 월드 구현 중에 어려움이 있어 질문드립니다.

1. 스카이박스가 움직이는 효과를 주려고 합니다. 평소에 유니티에서 아래와 같은 코드를 작성하여 카메라에 붙여 사용해왔습니다. 제페토에서 제공하는 타입스크립트에서 이와 같은 코드를 사용하려면 어떻게 고쳐야 하는 것인지 궁금합니다.
-------------------------------------------
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class RotateSky : MonoBehaviour
{
    public float RotateSpeed = 1.2f;
    

    void Update()
    {
        RenderSettings.skybox.SetFloat("_Rotation", Time.time * RotateSpeed);
    }
}
-------------------------------------------

2. 현실 시간이 흐름에 따라 스카이박스가 바뀌는 효과를 주려고 합니다. 낮->밤.
구현이 가능한 부분인지, 가능하다면 방법까지 알고 싶습니다.

**Comment by x4cf** [comment]
제페토 가이드 참고를 추천드리며, 서버에서 서버 시간을 가져와서 밤 낮을 구분할 수 있습니다
https://studio.zepeto.me/kr/guides/getting-started(world)
